packages/lodestar/src/chain/attestation.ts

```diff
@@ -97,19 +97,16 @@ export class AttestationProcessor implements IAttestationProcessor {
     const justifiedCheckpoint = this.forkChoice.getJustified();
     const currentSlot = this.forkChoice.headBlockSlot();
     const currentEpoch = computeEpochAtSlot(this.config, currentSlot);
-    let checkpointState: TreeBacked<BeaconState>;
-    if(justifiedCheckpoint.epoch > GENESIS_EPOCH) {
-      const justifiedBlock =
-        this.forkChoice.getBlockSummaryByBlockRoot(justifiedCheckpoint.root.valueOf() as Uint8Array);
-      if (justifiedBlock) {
-        checkpointState = (await this.db.stateCache.get(justifiedBlock.stateRoot)).state;
-      } else {
-        // should not happen
-        throw new Error(`Cannot find justified node of forkchoice, blockHash=${toHexString(justifiedCheckpoint.root)}`);
-      }
-    } else {
-      // should be genesis state
-      checkpointState = await this.db.stateArchive.get(0);
+    let stateCtx: ITreeStateContext;
+    const justifiedBlock = this.forkChoice.getBlockSummaryByBlockRoot(
+      justifiedCheckpoint.root.valueOf() as Uint8Array
+    );
+    if (justifiedBlock) {
+      stateCtx = await this.db.stateCache.get(justifiedBlock.stateRoot);
+    }
+    if (!justifiedBlock || !stateCtx) {
+      // should not happen
```

Comment:
is this comment meant to reinforce the idea that an error should be thrown under these conditions?  or is this also to maybe say "this shouldn't happen, but here's a sanity check just in case".  or both?  not a big deal, but just kinda caught my eye as being maybe a little ambiguous

Reply:
This is just a carry over from the existing impl. (the comment was already there)

I think this is a "this shouldn't happen, but here's a sanity check just in case" kind of situation tho.